### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by ZuhRa-HashiMi**
Use React Router.
Use React components.

### CODE REVIEW

File: src/components/navBar.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const NavBar = () => (
+  <header className="navbar">
+    <h1 className="nav_title">Math Magician</h1>
+    <nav className="links_list">
+      <Link to="/">Home</Link>
+      <Link to="/calculator">Calculator</Link>
+      <Link to="/quote">Quote</Link>
+    </nav>
+  </header>
```
Comment (mckent05) [inline src/components/navBar.js]: - Good job done so far 👍 👍, but there are some issues with your navbar, your navbar should contain 2 links, a link that displays a list of books, and another link `categories` that should render a page showing a button `check status`. Please ensure that your nav has links that render the desired page 🙏 
- Also please remove the Math magician header currently displaying on your page, this is an `Bookstore` project, please fix this 👍 

Your Project:
wtwtqhwhq

File: src/components/bookList.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SingleBook from './singleBook';
+
+const BookList = () => (
+  <div className="BookList">
+    <ul>
+      <SingleBook title="Alice In Wonderland" author="Lewis Carroll" />
+      <SingleBook title="Harry Porter" author="JK Rowlings" />
+      <SingleBook title="The Hobbit" author="JRR Tolkien" />
+    </ul>
```
Comment (mckent05) [inline src/components/bookList.js]: **OPTIONAL**-I suggest you do not need to use a `ul` tag here. Please fix this 🙏 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @ZuhRa-HashiMi, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- God job using the create-react-app command to initialize your project 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/navBar.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      <nav>
+        <ul>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/categories">Categories</Link>
+          </li>
+          <li>
+            <Link to="/profile">Profile</Link>
```
Comment (DammyShittu) [inline src/components/navBar.js]: - [ ] This profile link on the navbar is unnecessary as there is no component related to it. Kindly remove it as it is unneeded.

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import Footer from './components/footer';
+
+function App() {
+  return (
+    <div className="App">
+      <NavBar />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/categories" element={<Categories />} />
```
Comment (DammyShittu) [inline src/App.js]: - [ ] I am unable to navigate to the categories component. As can see I have clicked the categories navbar link yet I am still on the Homepage. Kindly fix this.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @ZuhRa-HashiMi,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MaganaAsiati — APPROVED** [review]
# Status: Approved [✔️](https://emojipedia.org/check-mark/)
## Hi @ZuhRa-HashiMi  , I'm @MaganaAsiati , your reviewer for this PR  :wave: 
Great work on making the changes requested by a previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 





## Highlights
- Descriptive pull request :heavy_check_mark:👨🏻‍💻
- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
Cheers and Happy coding! 🤓 👏👏👏

**Pull request merged**